### PR TITLE
mgr/dashboard: Fix missing root path of each session for CephFS

### PR DESCRIPTION
**Pull request by yongseokoh**
This PR allows the administrators to identify which root path each session has mounted. For instance, the FS tab in the dashboard can display the session id associated with the root path as shown in the attached picture. This can also be more useful in subvolume enviroment such as OpenStack Manila.

Fixes: https://tracker.ceph.com/issues/48062
Signed-off-by: Yongseok Oh <yongseok.oh@linecorp.com>

ceph_databoard_root_fix

## Checklist
- [x] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

File: src/pybind/mgr/dashboard/controllers/cephfs.py [hunk 1]
```diff
@@ -289,10 +289,12 @@ def _clients(self, fs_id):
...
                 client['version'] = client['client_metadata']['ceph_version']
                 client['hostname'] = client['client_metadata']['hostname']
+                client['root'] = client['client_metadata']['root']
             elif "kernel_version" in client['client_metadata']:  # pragma: no cover - no complexity
                 client['type'] = "kernel"
                 client['version'] = client['client_metadata']['kernel_version']
                 client['hostname'] = client['client_metadata']['hostname']
+                client['root'] = client['client_metadata']['root']
             else:  # pragma: no cover - no complexity there
                 client['type'] = "unknown"
```
Comment (votdev) [inline src/pybind/mgr/dashboard/controllers/cephfs.py:299]: I'm missing a default value like `client['root'] = None` or `client['root'] = ""` in this code path.

**Comment by votdev** [comment]
Please add a short summary to the PR description why it is useful to forward this field to the API consumer.

**Comment by yongseokoh** [comment]
I've modified the PR description. Please, review it. Thanks

**Comment by yongseokoh** [comment]
@votdev @batrick  Could please reassign the reviewer. I have mistakenly excluded the ceph/api in the reviewer list.

**Review by epuertat — APPROVED** [review]
Thanks a lot @yongseokoh for your contribution!

**Review by nizamial09 — APPROVED** [review]

**Comment by epuertat** [comment]
jenkins retest this please

**Pull request merged**